build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     google()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:8.3.0'
+    classpath 'com.android.tools.build:gradle:8.4.0'
```

Comment:
When I try to run it using the IDE I get this error:
`Minimum supported Gradle version is 8.6. Current version is 8.4.`

When using the `make` it works fine. Are there many changes if we upgrade to 8.6?

Reply:
Not that I know of, it should "just work". I tried it again from scratch by re-cloning the repo and running it using the IDE and it worked fine.
It's probably specific to gradle <-> android studio, I'm not very familiar with it but could there be an integration to upgrade on your end? Have you tried running "Sync Project with Gradle Files"?

Reply:
I made a fresh install and now it's good, something dirty or old